src/common/scoped_message_writer.h

```diff
@@ -91,7 +91,7 @@ class scoped_message_writer
     {
       m_flush = false;
 
-      MCLOG(m_log_level, "msgwriter", m_oss.str());
+      MCLOG_FILE(m_log_level, "msgwriter", m_oss.str());
 
       if (epee::console_color_default == m_color)
```

Comment:
They're meant to be console output, so still showing up when all logs are disabled, as happens with monero-wallet-cli, where all logs go to file and not console. monerod doesn't work that way, and indeed we rely on logs to give important info, but ideally it would (and it can, if you set logs to none). In that case, you still get command output.